DependencyInjection/FOSUserExtension.php

```diff
@@ -207,4 +211,12 @@ protected function remapParametersNamespaces(array $config, ContainerBuilder $co
             }
         }
     }
+
+    protected function setForceLoginOnPasswordChange(ContainerBuilder $container)
+    {
+        foreach ($container->findTaggedServiceIds('fos_user.user_provider') as $id => $tags) {
```

Comment:
finding tagged services here does not make sense: it only allows to find services defined in the FOSUserBundle service files anyway, so we don't need a tag